Sources/Basic/OrderedSet.swift

```diff
@@ -13,7 +13,11 @@
 public struct OrderedSet<E: Hashable>: Equatable, Collection {
     public typealias Element = E
     public typealias Index = Int
+    #if swift(>=4.2)
+    public typealias Indices = Range<Int>
+    #else
     public typealias Indices = CountableRange<Int>
+    #endif
```

Comment:
@jrose-apple is this check enough to not break compatibility with 4.0?

Reply:
Should be fine, but shouldn't that be checking for 4.1, not 4.2?